hack/setup-multi-tenancy/setup_client.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2020 Authors of Arktos.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+if [ "$#" -ne 3 ]
+then
+        echo "Usage: setup_client.sh [tenant] [person] [ip]"
+        exit 1
+fi
+
+tenant=$1
+person=$2
+namespace="default"
+tenant_person="${tenant}-${person}"
+group_name="${tenant}-group"
+context_name="${tenant_person}-context"
+ip=$3
```

Comment:
can we make this parameter optional? The default value could be "localhost:6443", as a lot of engineers are running test cluster from the local dev box.

Reply:
good idea.

Reply:
added